stytch/src/main/kotlin/com/stytch/java/b2b/api/sessions/Sessions.kt

```diff
@@ -519,14 +519,13 @@ internal class SessionsImpl(
         authorizationCheck: AuthorizationCheck?,
     ): StytchResult<MemberSession?> =
         withContext(Dispatchers.IO) {
-            try {
-                authenticateJwtLocal(jwt = jwt, maxTokenAgeSeconds = maxTokenAgeSeconds, authorizationCheck = authorizationCheck)
-            } catch (e: JWTException) {
-                when (val result = authenticate(AuthenticateRequest(sessionJwt = jwt, authorizationCheck = authorizationCheck))) {
-                    is StytchResult.Success -> StytchResult.Success(result.value.memberSession)
-                    else -> StytchResult.Success(null)
-                }
+          when (val localResult = authenticateJwtLocal(jwt = jwt, maxTokenAgeSeconds = maxTokenAgeSeconds, authorizationCheck = authorizationCheck)) {
+            is StytchResult.Success -> StytchResult.Success(localResult.value.memberSession)
```

Comment:
should be `StytchResult.Success(JWTSessionResponse(localResult.value))` (ie: wrapped in a `JWTSessionResponse`, and the `value` _is_ the session)

Reply:
Since B2B sessions didn't ship that way, we'd have to do a MAJOR version bump now if we made that change. Since we just bumped the MAJOR version, I think we should leave that until the next time we need to make major breaking changes.

Reply:
Ahhhhh, yeah, good call. Can you make a ticket to update the B2B methods for the next major bump? Don't wanna forget it

Reply:
We do still just want to return `localResult.value` though, since that's already the memberSession